### DISCUSSION TITLE
文章限制阅读 Pro功能建议

### DISCUSSION THREAD

**Discussion by byteforgecn** [halo-dev/halo#5840] [Q&A]
### 你当前使用的版本

2.14.1 插件版本1.2.3

### 描述一下此特性

1. 公众号验证模式下, 个别用户会看错或者输错文章页面提示要求输入的4位数, 此时公众号无响应, 是预期的 但是用户并不知道为什么, 希望能够给比较粗心的用户一个提示 您输入的4位数不匹配等语句, 考虑到每个人需求都不一样有些用户可能不喜欢这个提示, 可以做成开关, 控制程序是否会提醒用户输入有误
2. "首次访问，需要验证" 这个提示并不合适, 现在插件逻辑是退出登录或过一段时间可能这个验证码就失效了, 那么就需要重新去公众号发送此验证码, 用户反馈原文是: "你好, 请问一下是不是我的浏览器有问题? 我昨天以及验证访问过了今天还说我首次访问需要验证",  简单改这个提示为类似"本次访问需要验证码"的就行. 

### 附加信息

关于第二点的补充
_如果实力允许希望可以开关一个类似频繁模式的开关,他标识了一个用户只需一次验证还是会像现在这样经常会过期以确保不取关公众号_

**Comment by ruibaby** [comment]
感谢反馈这个问题，但这个插件属于应用市场的付费插件，在 Halo 的源代码仓库中可能无法解决，所以我将转移到 Disscussion 中。

**Comment by ruibaby** [comment]
cc @liuchangfitcloud 有时间帮忙看看这个这个问题。

**Comment by Arch4AI** [comment]
是否需要在基础上集成部分的文本自动回复？

**Reply by byteforgecn** [reply]
抱歉,没理解您具体意思

**Comment by CF673** [comment]
是否考虑内置原生评论查看，密码查看。靠插件实现始终感觉怪怪的……
